NEWS.txt

```diff
@@ -1,4 +1,6 @@
 Version 7.27 - not yet released
+* user interface
+  - rasp sources from repository
```

Comment:
NEWS changes should be part of the commit implementing the feature; that way, a "git blame" on NEWS.txt easily reveals the commit intruducing the change.